docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
     # Comment to use dockerhub image
     build: .
     image: pixelfed
+    restart: always
```

Comment:
perhaps `restart: unless-stopped` instead? this way it'll stop if asked.

Reply:
I'm happy with that.